lib/link_checker/uri_checker/http_checker.rb

```diff
@@ -290,5 +289,9 @@ def additional_connection_headers
 
       { 'Rate-Limit-Token': Rails.application.secrets.rate_limit_token }
     end
+
+    def use_google_safebrowsing?
+      Rails.env.production? && Rails.application.secrets.google_api_key
```

Comment:
This should be an `||`.

Reply:
I was going down the production only route, but makes sense if you want to test locally.

Reply:
Done @thomasleese